pwa/src/styles/main.css

```diff
@@ -219,3 +219,8 @@ a:hover {
     border: none !important;
     background: inherit !important;
 }
+
+#helpModal {
+    text-align: left;
+    color: black;
+}
```

Comment:
Usually we reserve using an `id` (`#`-selector) for edge case javascript selecting (it was very popular in things like `jQuery` and is still in use in things like `Bootstrap` so that's why we need to actually add the `#helpModal` id at this moment).

In React however, we pretty much never want to add an `id` to a component. Therefore, we also would like to not use the `#` selector in our `css`.

Above being said, usually we would have a unique `className` for each component and we can scope our styling like that. However, because of the way everything's structured right now, that would be a bit of a rework.

Therefore: I would like to suggest to just swap this `#helpModal` with `.modal` 👍🏼